shared/constants/snaps/permissions.ts

```diff
@@ -26,6 +26,8 @@ export const ExcludedSnapPermissions = Object.freeze({
 });
 
 export const ExcludedSnapEndowments = Object.freeze({
+  'endowment:page-home':
+    'This permission is still in development and therefore not available.',
   ///: BEGIN:ONLY_INCLUDE_IN(build-main)
   'endowment:lifecycle-hooks':
```

Comment:
Should we enable this?

Reply:
Will enable in a follow up PR. The follow up PR will add the permission copy and the actual feature integration.